assets/src/edit-story/components/library/tabs/index.js

```diff
@@ -6,51 +6,70 @@ import styled from 'styled-components';
 /**
  * Internal dependencies
  */
+import { Navigable, NavigableGroup } from '../../focusable';
 import MediaIcon from './media.svg';
 import TextIcon from './text.svg';
 import ShapesIcon from './shapes.svg';
 import LinksIcon from './links.svg';
 
-const Tabs = styled.ul`
+const Tabs = styled( NavigableGroup ).attrs( {
+	element: 'nav',
+	role: 'tablist',
+	tabindex: -1,
```

Comment:
Should the default focus CSS (outline) also be removed here? (Since it can still focus by clicking)